src/adaptermanager.js

```diff
@@ -8,71 +9,71 @@ var events = require('./events');
 var _bidderRegistry = {};
 exports.bidderRegistry = _bidderRegistry;
 
-exports.callBids = function (bidderArr) {
-  for (var i = 0; i < bidderArr.length; i++) {
-    //use the bidder code to identify which function to call
-    var bidder = bidderArr[i];
-    if (bidder.bidderCode && _bidderRegistry[bidder.bidderCode]) {
-      utils.logMessage('CALLING BIDDER ======= ' + bidder.bidderCode);
-      var currentBidder = _bidderRegistry[bidder.bidderCode];
-
-      //emit 'bidRequested' event
-      events.emit(CONSTANTS.EVENTS.BID_REQUESTED, bidder);
-      currentBidder.callBids(bidder);
-
-      // if the bidder didn't explicitly set the number of bids
-      // expected, default to the number of bids passed into the bidder
-      if (bidmanager.getExpectedBidsCount(bidder.bidderCode) === undefined) {
-        bidmanager.setExpectedBidsCount(bidder.bidderCode, bidder.bids.length);
-      }
-
-      var currentTime = new Date().getTime();
-      bidmanager.registerBidRequestTime(bidder.bidderCode, currentTime);
-
-      if (currentBidder.defaultBidderSettings) {
-        bidmanager.registerDefaultBidderSetting(bidder.bidderCode, currentBidder.defaultBidderSettings);
-      }
+exports.callBids = function(bidderArr) {
+	for (var i = 0; i < bidderArr.length; i++) {
```

Comment:
Please use 2 spaces for indenting. It looks like your editor modified this file inadvertently.